src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -24,7 +28,10 @@ public Order completeOrder(Long userId) {
         Product macBook = new Product("MacBook Air 2020", BigDecimal.valueOf(1399));
         Product xiaomi = new Product("Xiaomi 12", BigDecimal.valueOf(499));
         List<Product> products = List.of(iphone, macBook, xiaomi);
-        // TODO: add log message about successful fetched data from DB
+        logger.info("successful fetched data from DB. Products are:");
+        for (Product product : products) {
```

Comment:
You can replace it with "products.toString()"
logger.info("successful fetched data from DB. Products are:  {}", products.toString());

Reply:
Yes, but it will be print in one line. And if we will have many prodacts?

Reply:
Lets use Stream Api) And lets add info about user fetching the data (userId)